packages/clima_ui/analysis_options.yaml

```diff
@@ -38,3 +38,6 @@ linter:
 
     # Very infuriating and unnecessary
     sort_pub_dependencies: false
+
+    # Still in beta as of v1.6.0
```

Comment:
Version v1.6.0 of what?

Reply:
Of Lint, but you reminded me. I forgot to check if it's out in beta now in v1.7.1. Can you check it out for me please?

Reply:
You're misunderstanding; the rule in general is beta, nothing to do with the `lint` package.